test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -409,6 +409,7 @@ func benchmarkScheduling(numNodes, numExistingPods, minPods int,
 		// Since the total amount of time is relatively large, it might not be a concern.
 		time.Sleep(100 * time.Millisecond)
 	}
+	b.StopTimer()
```

Comment:
LGTM. Let's put a comment explaining this.

Reply:
LGTM @draveness 
Agree with @Huang-Wei . For readability, it is necessary to note why this line was added :)

Reply:
done, PTAL